demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime-grpc.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: caikit-tgis-runtime-grpc
+spec:
+  multiModel: false
+  supportedModelFormats:
+    # Note: this currently *only* supports caikit format models
+    - autoSelect: true
+      name: caikit
+  containers:
+    - name: kserve-container
+      image: quay.io/opendatahub/text-generation-inference:stable
+      command: ["text-generation-launcher"]
+      args: ["--model-name=/mnt/models/artifacts/"]
+      env:
+        - name: TRANSFORMERS_CACHE
+          value: /tmp/transformers_cache
+      # resources: # configure as required
+      #   requests:
+      #     cpu: 8
+      #     memory: 16Gi
+    - name: transformer-container
+      image: quay.io/opendatahub/caikit-tgis-serving:stable
```

Comment:
```suggestion
      image: quay.io/opendatahub/caikit-tgis-serving:stable
      command: ["python", "-m", "caikit.runtime.grpc_server"]
```

Reply:
Since we'll only be exposing one port, I see no reason to bring both servers up

Reply:
@dtrifiro @Xaenalt   I added (still not pushed) the command: ["python", "-m", "caikit.runtime.grpc/http_server"]  line in  caikit-tgis-serving-runtime-grpc.yaml and caikit-tgis-serving-runtime.yaml respectively.

The inference invocations work with this added container command for both protocols.

I could push the code since it has been tested successfully however I must concede I did not understand why this addition was suggested (how does it help?).  Tx for help on this

Reply:
`python -m caikit.runtime` spins up both grpc and http servers. 

Since we're only going to be using either `grpc` or `http` (cannot expose two ports with kserve), we can skip bringing up both servers and just using the relevant command for each manifest:

- `python -m caikit.runtime.http_server` for http
- `python -m caikit.runtime.grpc_server` for grpc

Reply:
@dtrifiro Ok thanks.  I pushed the change to the code